docs/tutorials/multimodal/convert_data_to_coco_format.md

```diff
@@ -0,0 +1,169 @@
+Below are the key names and value definitions inside ``*_labels.json``:
+
+```javascript
+{
+    "info": info,
+    "licenses": [license], 
+    "images": [image],  // list of all images in the dataset
+    "annotations": [annotation],  // list of all annotations in the dataset
+    "categories": [category]  // list of all categories
+}
+
+where:
+
+info = {
+    "year": int, 
+    "version": str, 
+    "description": str, 
+    "contributor": str, 
+    "url": str, 
+    "date_created": datetime,
+}
+
+license = {
+    "id": int, 
+    "name": str, 
+    "url": str,
+}
+
+image = {
+    "id": int, 
+    "width": int, 
+    "height": int, 
+    "file_name": str, 
+    "license": int,  // the id of the license
+    "date_captured": datetime,
+}
+
+category = {
+    "id": int, 
+    "name": str, 
+    "supercategory": str,
+}
+
+annotation = {
+    "id": int, 
+    "image_id": int,  // the id of the image that the annotation belongs to
+    "category_id": int,  // the id of the category that the annotation belongs to
+    "segmentation": RLE or [polygon], 
+    "area": float, 
+    "bbox": [x,y,width,height], 
+    "iscrowd": int,  // 0 or 1,
+}
+```
+For the sole purpose of running AutoMM, the fields ``"info"`` and ``"licenses"`` are optional. 
+``"images"``, ``"categories"``, and ``"annotations"`` are required.
+
+
+
+```json
+{
+  "info": {...},
+"licenses": [
+        {
+            "url": "http://creativecommons.org/licenses/by-nc-sa/2.0/", 
+            "id": 1, 
+            "name": "Attribution-NonCommercial-ShareAlike License"
+        },
+        ...
+    ],
+    "categories": [
+        {"supercategory": "person", "id": 1, "name": "person"},
+        {"supercategory": "vehicle", "id": 2, "name": "bicycle"},
+        {"supercategory": "vehicle", "id": 3, "name": "car"},
+        {"supercategory": "vehicle", "id": 4, "name": "motorcycle"},
+        ...
+    ],
+        
+    "images": [
+        {
+            "license": 4, 
+            "file_name": "<imagename0>.<ext>", 
+            "height": 427, 
+            "width": 640, 
+            "date_captured": null, 
+            "id": 397133
+        },
+        ...
+    ],
+    "annotations": [
+        
+        ...
+    ]
+}
+```
+
+
+
+
+
+
+
+The following is an example of one sample annotated with COCO format
+
+## Converting Pascal VOC dataset to MSCOCO format
+Pascal VOC is another very popular dataset for object detection, and it also follows its own data format. 
+We provide an example script to easily convert VOC dataset to COCO format.
+
+The script can be found in the following directory:
+```
+./examples/automm/object_detection/voc2coco.py
+```
+
+Assuming your VOC dataset has the following structure
+```
+<path_to_VOCdevkit>/
+    VOC2007/
+        ...
+    VOC2012/
+        ...
+    ...
+```
+
+#### 1. Go to the script folder:
+```
+cd ./examples/automm/object_detection/
+```
+
+#### 2. Run the following command:
+```
+python voc2coco.py <path_to_VOCdevkit> \
```

Comment:
Change to use the voc2coco.py in our example.

Reply:
Addressed in ce884dba67c46572c8b5c699980d4cfebf55cdd0